src/LogoFX.Client.Core.Platform/src/PlatformDispatch.cs

```diff
@@ -1,16 +1,28 @@
-using System.Windows.Threading;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
@glyad Please remove - no need to add dependency on Resharper here

Reply:
Fixed